### ISSUE TITLE
Dragons Dogma 2 works, how to inside.

### ISSUE THREAD

**Issue by TimmyP777** [Nukem9/dlssg-to-fsr3#406]
Update game on steam (small patch on 3/22).

Download this mod,

"DLSS3 enabler" for DD2 by Puredark (free), 

and the DLSS3.6 dlls.

Copy all next to DD2 exe. Make sure you copy 3.6 dll last to overwrite the others.
Enjoy.

**Comment by TimmyP777** [comment]
PS: if you start to get loading crashes, delete the cache file inside the game directory. Will fix. Will not need to recompile shaders.

**Comment by skeycles** [comment]
yeah....it crashes, but what's the exact file I should delete? cuz I see no cache file inside the game directory..

**Comment by TimmyP777** [comment]
Sorry man they banned me from the Steam forums for 1 month for bringing this up. I did NOTHING. Sorry but Im not in the mood to provide any help rn. Just wait for FSR3.1.

**Comment by skeycles** [comment]
WHAT? they ban you just for this? but yeah..i feel you... its okay

**Comment by TimmyP777** [comment]
Yeah 1 month ban.

I advise anyone never to say things like "this will turn your 3000 into a slower 4000" and **** like that. This is so... corrupt? I know there is more to this. A month! I just bought it!

Everyone on that forum is bashing the game, and I am trying to help people get more performance with this mod... and they ban me? Let me sleep on it at least. Unreal.

**Comment by Name1100** [comment]
You guys got the HUD/UI flashing issue? I got it working but even the subtitles keep flashing.

**Comment by Aidorin0** [comment]
Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.

**Comment by skeycles** [comment]
> Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.

wait, how the hell you guys did it? I got a 3070 laptop cpu and used it in skyrim and 2077, but follow these steps in DD2 I just keep on crashing at start

**Comment by Aidorin0** [comment]
> > Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.
> 
> wait, how the hell you guys did it? I got a 3070 laptop cpu and used it in skyrim and 2077, but follow these steps in DD2 I just keep on crashing at start

Well, It turns out that It always crashes if the cache file is inside game directory. So you have to delete the cache file before launching the game. And because the game creates the cache file everytime you launch it, you previously must delete the cache each time you launch the game.

**Comment by skeycles** [comment]
> > > Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.
> > 
> > 
> > wait, how the hell you guys did it? I got a 3070 laptop cpu and used it in skyrim and 2077, but follow these steps in DD2 I just keep on crashing at start
> 
> Well, It turns out that It always crashes if the cache file is inside game directory. So you have to delete the cache file before launching the game. And because the game creates the cache file everytime you launch it, you previously must delete the cache each time you launch the game. I hope we can get a workaround to this cache file issue.

right...but what version of DLSS-FSR file you are using? My good gentleman, the god damn frame rate is killing me!

**Comment by Aidorin0** [comment]
> > > > Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.
> > > 
> > > 
> > > wait, how the hell you guys did it? I got a 3070 laptop cpu and used it in skyrim and 2077, but follow these steps in DD2 I just keep on crashing at start
> > 
> > 
> > Well, It turns out that It always crashes if the cache file is inside game directory. So you have to delete the cache file before launching the game. And because the game creates the cache file everytime you launch it, you previously must delete the cache each time you launch the game. I hope we can get a workaround to this cache file issue.
> 
> right...but what version of DLSS-FSR file you are using? My good gentleman, the god damn frame rate is killing me!

I'm using Nukem's the last one, v0.9, and the "version.dll' version. Obviously with Puredark's FG enabler mod. You don't need to replace the DLSS dll with 3.6 version as OP suggest, It also works with game's default v3.5.10.

**Comment by TimmyP777** [comment]
> > > Yeah, it works, but it has that HUD flashing/flickering, and it crashes quite often. But it's a promising start.
> > 
> > 
> > wait, how the hell you guys did it? I got a 3070 laptop cpu and used it in skyrim and 2077, but follow these steps in DD2 I just keep on crashing at start
> 
> Well, It turns out that It always crashes if the cache file is inside game directory. So you have to delete the cache file before launching the game. And because the game creates the cache file everytime you launch it, you previously must delete the cache each time you launch the game. I hope we can get a workaround to this cache file issue.

Here is an easy temp workaround. Make a new batch (.bat) file DD2 exe directory and put this inside (3 lines):

del *.cache*
timeout -t 1
start DD2.exe

Run the script instead of the exe. It will delete the file if it exists, wait 1 second, then start the game.

**Comment by TadeuAriasVillares** [comment]
Guys, anyone has a workaround for the UI/subtitles flickering?

**Comment by TimmyP777** [comment]
^There wont be a fix before FSR3.1 is implemented so just wait for that. 

PS This game\mod is very sensitive to virtual memory. Ive you have paging manually set to any level, I would switch it to system managed!

**Comment by TadeuAriasVillares** [comment]
^Thanks. It will be sooner than we think thou --> https://videocardz.com/newz/amd-fsr-3-1-update-to-enable-frame-generation-for-other-upscaling-technologies

**Comment by TimmyP777** [comment]
Has anyone gotten this to work today? It worked fine yesterday, but now I cant get it once.

**Comment by Hinkiii** [comment]
Universal showing greyed out on a GTX card sadly

**Comment by Sicara9** [comment]
This works perfectly fine for me, but I did not have to delete my cache. My game was crashing until i disabled Nvidia overlay.

**Comment by TimmyP777** [comment]
Sometimes you dont need to delete cache, if it freezes at Capcom logo you know what to do.

**Comment by ImaniiTy** [comment]
disabling Nvidia Overlay fixed the crashes for some people.

**Comment by TimmyP777** [comment]
> disabling Nvidia Overlay fixed the crashes for some people.

Same with Nvidia app. I didn't even open it, just changed driver versions.

**Comment by Aidorin0** [comment]
Using Reframework seems to fix the cache file crash, so no more need to delete It each time we launch the game. You can get the last one from this GitHub link, download the DD2.zip and put the dinput8.dll file (you only need this one) into game folder. Start the game once so Framework creates the '_storage_' folder.

Then just move the dlssg_to_fsr_amd_is_better.dll file from Nukem's mod (but not the version.dll one) to the _storage_ folder created by Reframework.

https://github.com/praydog/REFramework-nightly/releases

**Comment by GOVTZILLA** [comment]
I installed the mod, everything seems to be working, but the color of the picture has changed... How to fix it?
When I press alt+tab the color becomes normal.
![изображение_2024-03-31_033933133](https://github.com/Nukem9/dlssg-to-fsr3/assets/165505131/6ec2e10e-664c-40bd-80bb-4680d4c12bdb)
![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/165505131/5bae595a-a149-4a6f-992c-6e1fe42d55c2)
![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/165505131/e48ec7bc-38c5-4aba-a7ea-8271fd42adec)

**Comment by TimmyP777** [comment]
I think that happens on older version? There is a thread about Cyberpunk, but I think I  might fixed that fiddling with HDR settings.

**Comment by jmmontoro** [comment]
Anyone else noticing a lot more UI artifacts with the RE Framework method?

**Comment by Aidorin0** [comment]
> Anyone else noticing a lot more UI artifacts with the RE Framework method?

I don't see any difference.

**Comment by TadeuAriasVillares** [comment]
^ Using RE Framework stops the UI and subtitles flickering issue?

**Comment by blooest** [comment]
No. But I can't imagine why it would cause them to be any worse. That's a product of the UI being tweened along with the game because FSR in its current implementation, at least through this method, doesn't have the temporal understanding that they're _not_ a part of the world.

**Comment by Dustyse** [comment]
YMMV, but I fixed the UI flickering by just enabling FSR3 instead of DLSS in the game's settings. The FG still works. Although sometimes the game might refuse to start the next time if I don't remember to set it back to DLSS before exiting. Deleting the cache in that case fixes it.

**Comment by TimmyP777** [comment]
The point of this mod is to use dlss upscaling and fsrfg at the same time, to "enable" dlss3.

Using fsr3 does not allow you to use dlss upscaling, so your image will look notably worse.
Fsr 3.1 changes this.

**Comment by Dustyse** [comment]
Ah, right, I was under the impression it didn't matter which one you had enabled but that's wrong obviously.

Though to be fair, the image quality drop isn't that drastic to my eyes at least. I only have a 1440p screen though. Still, I'll take the image quality hit for now so I don't drive myself mad with the flickering UI.

**Comment by jmmontoro** [comment]
New update breaks the mod, it shouldn't be necessary to use PureDark anymore, this mod enables the option but the game crashes instantly if activated.
